Failure building `build-recorder` with automated build & packaging `xbps-src` tool.

Title basically.

It seems to be a "relative path" issue, how do we tackle it?

Perhaps introduce an `include` dir and including project-specific headers that way?
 
The error:

> cc -DHAVE_CONFIG_H -I.     -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe    -ffile-prefix-map=/builddir/build-recorder-1.0=. -MT tracer.o -MD -MP -MF $depbase.Tpo -c -o tracer.o tracer.c &&\
> mv -f $depbase.Tpo $depbase.Po
> record.c:9:17: fatal error: record.h: No such file or directory
>     9 | #include        "record.h"
>       |                 ^~~~~~~~~~
> compilation terminated.
> make[2]: *** [Makefile:384: record.o] Error 1
> make[2]: *** Waiting for unfinished jobs....
> main.c:17:10: fatal error: record.h: No such file or directory
>    17 | #include "record.h"
>       |          ^~~~~~~~~~
> compilation terminated.
> make[2]: *** [Makefile:384: main.o] Error 1
> tracer.c:27:17: fatal error: types.h: No such file or directory
>    27 | #include        "types.h"
>       |                 ^~~~~~~~~
> compilation terminated.
> make[2]: *** [Makefile:384: tracer.o] Error 1
> hash.c:34:17: fatal error: hash.h: No such file or directory
>    34 | #include        "hash.h"
>       |                 ^~~~~~~~
> compilation terminated.
> make[2]: *** [Makefile:384: hash.o] Error 1
> make[2]: Leaving directory '/builddir/build-recorder-1.0/src'
> make[1]: *** [Makefile:270: all] Error 2
> make[1]: Leaving directory '/builddir/build-recorder-1.0/src'
> make: *** [Makefile:344: all-recursive] Error 1
> => ERROR: build-recorder-1.0_1: do_build: '${make_cmd} ${makejobs} ${make_build_args} ${make_build_target}' exited with 2
> => ERROR:   in do_build() at common/build-style/gnu-configure.sh:15
>

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -12,8 +12,11 @@ build_recorder_SOURCES = \
 	tracer.c \
 	$(EMPTY)
 
+dist_noinst_HEADERS = *.h
+
 SCHEMA = ../doc/build-recorder-schema.ttl
 
+dist_data_DATA = $(SCHEMA)
 
 if HAVE_XXD
 schema.c: $(SCHEMA)

